src/SlmQueue/Queue/AbstractQueue.php

```diff
@@ -65,6 +65,14 @@ public function unserializeJob($string, array $metadata = array())
         $metadata += $data['metadata'];
         $content  =  unserialize($data['content']);
 
+        if (isset($data['metadata']['__jobchain__'])) {
+            $self = $this; // needed for php5.3 (?)
```

Comment:
If we emrge this, it'll be part of 0.5, and I think it's time to set 5.4 as min dependency.